core/domain/topic_services.py

```diff
@@ -724,6 +725,14 @@ def delete_topic(committer_id, topic_id, force_deletion=False):
     for subtopic in topic_model.subtopics:
         subtopic_page_services.delete_subtopic_page(
             committer_id, topic_id, subtopic['id'])
+
+    all_story_references = (
+        topic_model.canonical_story_references +
+        topic_model.additional_story_references)
+    for story_reference in all_story_references:
```

Comment:
Ditto -- delete_multi

Reply:
delete_story has more functionalities in it (like deleting the expContextModel, delete story summary and delete cache). We'll have to duplicate all of that here for this...

Reply:
Ah ok -- I think this will happen rarely enough that leaving it as-is is probably OK. Thanks for pushing back.